keyboards/handwired/42/halconf.c

```diff
@@ -0,0 +1,524 @@
+/*
```

Comment:
This file won't be used as it's called `halconf.c`, whereas it should be `halconf.h`.

If you're only using this to enable I2C, you can do this instead of replicating the whole file:

```c
<license header>
#pragma once
#define HAL_USE_I2C TRUE
#include_next "halconf.h"
```

Reply:
@tzarc It may be a good idea to have this posted somewhere in the docs.